genie-web/src/test/java/com/netflix/genie/web/controllers/ControllerUtilsUnitTests.java

```diff
@@ -77,4 +77,25 @@ public void canGetRemainingPath() {
             .thenReturn("/api/v3/jobs/{id}/output/**");
         Assert.assertThat(ControllerUtils.getRemainingPath(request), Matchers.is("stdout"));
     }
+
```

Comment:
Maybe test corner cases just to freeze the behavior?
i.e. empty 'path', request not containing path, ...

Reply:
Added another commit with some improved functionality